lib/utils/connect.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
```

Comment:
```suggestion
const { parse } = require('url');
const mongoose = require('mongoose');

function redact(uri) {
  const parsedUri = parse(uri);
  const authPart = parsedUri.auth ? '****:****@' : ''
  return `${parsedUri.protocol}://${authPart}${parsedUri.hostname}:${parsedUri.port}${parsedUri.path}`
}

function log(event, dbUri) {
  return function () {
    console.log(`Connection ${event} on ${redact(dbUri)}`);
  }
}
```
Here is a suggestion from Ryan's class example.